### PR TITLE
Drop 404d Ricin website.

### PR DESCRIPTION
**Pull request by robinlinden**

### CODE REVIEW

**Comment by anoadragon453** [comment]
rip

**Comment by Zer0-One** [comment]
rip

**Comment by nurupo** [comment]
@SkyzohKey 

1. Is Konv still being developed or is it abandoned?

2. Is Konv a drop-in replacement for Ricin, i.e. a user using Ricin can install Konv and have all their settings from Ricin (proxy settings, tcp/udp, tox profiles, etc.) automaically picked up by Konv?

3. Which toxcore does Konv use: irungentoo's or toktok's?

**Comment by ProMcTagonist** [comment]
F

**Comment by SkyzohKey** [comment]
@nurupo 

1. Yep, Konv is still being developed, it's just paused as I'm in job search process ATM. I'll work on it again once I have found a job ;)
2. Konv should have an import option to migrate a Ricin profile + it's settings, yep.
3. It uses TokTok's one. I still have some work to do on the binding before being able to plug the interface to ToxCore.

**#RIP** Ricin.im, **#Hello** [Konv.im](https://konv.im)

**Comment by anoadragon453** [comment]
@SkyzohKey Very nice website, btw.

**Comment by tox-user** [comment]
ricin.im is still dead and still haven't been removed. I think it would be best to remove it as soon as possible, beucase we don't know what happens to that domain and who is going to own it.

**Comment by SkyzohKey** [comment]
I'll try to renew the domain name ASAP even if it isn't in use anymore. For the sake of security. But yes ofc it should be dropped. :)

File: themes/website/templates/clients.html [hunk 1]
```diff
@@ -82,9 +82,6 @@ <h2 class="header">Ricin</h2>
 
 				<table class="table table-condensed">
 					<tbody>
-						<tr>
-							<th>Website:</th>
-							<th><a href="https://ricin.im">https://ricin.im</a>
 						<tr>
```
Comment (SkyzohKey) [inline themes/website/templates/clients.html]: Instead of removing the URL, maybe change it to https://konv.im as it stills provides valid downloads?

**Comment by tox-user** [comment]
If you were planning to renew the website why change the url to konv.im?

**Comment by nurupo** [comment]
Konv is a new client, different from Ricin. SkyzohKey has written one client, then dropped it to write a new one. Written a new client and then dropped it to write yet another new one... I think he has written 3 or so Tox clients now, 2 of which are dropped -_-

Also, if I'm not mistaken, konv.im previously pointed to a website about Konv client, but for some reason it now talks about Ricin.

Earlier in this thread I was talking about replacing Ricin with Konv, since Ricin was dropped, but I guess Konv got dropped and Ricin continues? I'm very confused.

**Comment by nurupo** [comment]
I can change the URL to point to konv.im, as long as the website remains being about Ricin, not Konv.

**Comment by tox-user** [comment]
If Konv continues to be developed, maybe it would be better to eventually drop Ricin from the site completely? We probably shouldn't "advertise" an outdated, abandoned client for long.

**Comment by nurupo** [comment]
Right, but like I said, it looks like Ricin is being developed but Konv is abandoned now?

[Ricin had last commit among all branches in July](https://github.com/RicinApp/Ricin) but [Konv had last commit in May](https://github.com/SkyzohKey/Konv). Huh, now it sounds like they both are abandoned.

**Comment by nurupo** [comment]
Oh, right, I remember now, SkyzohKey is working on a new client https://github.com/Tox-Client/client. So, should we remove Ricin from the website and package repository, making a blog post about it?

**Comment by tox-user** [comment]
I think we should, but perhaps we should wait for an answer from @SkyzohKey first.

**Comment by tox-user** [comment]
Btw Tox-Client is not a confusing name at all ;)

**Comment by SkyzohKey** [comment]
@nurupo Ricin **IS** definitly dead and unmaintained, even it's still running fine. I just merged Groupchats sometimes ago for people still using it ^^. Konv was a rewrite of Ricin to make the code better and easier to test/maintain. But then I discovered React/React Native and jumped on the wagon to make Tox-Client.

@tox-user Tox-Client (code name Konv, yeah that's confusing I know, xD) is yet in a really alpha stage and not even usable as a Tox client. But my main goal here is to **unify** Tox usage on every platform (device agnostic) so the Tox-client name seemed better and less long that Tox-Universal-Client xD.

I'm the king of confusion. May write a blog post for Tox.chat eventualy if wanted to clarify things.

**Comment by nurupo** [comment]
Ok, that explains a lot.

So what I gather is that we should just remove Ricin without replacing it with anything. We might add Tox-Client at a later time, once it matures enough and we would be sure @SkyzohKey isn't going to start another new client :D

**Comment by SkyzohKey** [comment]
I'd prefer to let the website on the list because downloads are still available and Ricin (hosted at konv) is still operating fine with Tox network. 

Any way, I'm planning to rename Tox-Client to konv soon and make a nice website for it (or maybe to merge it with the [beta.tox.chat](http://beta.tox.chat) one :p)

**Pull request closed without merging**